rules/rules-reviewed/technology-usage/database.windup.groovy

```diff
@@ -49,39 +48,39 @@ ruleSet("database")
             perform(event, context, payload.getFile(), "HSQLDB Driver", true)
         }
     })
-    .withId(String.format("database-0%d00", id++))
+    .withId("database-01400")
     .addRule()
     .when(File.inFileNamed("{*}mysql-connector{*}.jar"))
     .perform(new AbstractIterationOperation<FileLocationModel>() {
         void perform(GraphRewrite event, EvaluationContext context, FileLocationModel payload) {
             perform(event, context, payload.getFile(), "MySQL Driver", true)
         }
     })
-    .withId(String.format("database-0%d00", id++))
+    .withId("database-01500")
     .addRule()
     .when(File.inFileNamed("{*}derby{*}.jar"))
     .perform(new AbstractIterationOperation<FileLocationModel>() {
         void perform(GraphRewrite event, EvaluationContext context, FileLocationModel payload) {
             perform(event, context, payload.getFile(), "Derby Driver", true)
         }
     })
-    .withId(String.format("database-0%d00", id++))
+    .withId("database-01600")
```

Comment:
For postgresql, you defined the rule with id database-01600 but override by database-01700?